analytics/src/views/Graphs/components/Filters/index.js

```diff
@@ -1 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore, combineReducers } from 'redux';
+import { Provider } from 'react-redux';
+import { reducer as formReducer } from 'redux-form';
+import App  from './Filters';
+import * as serviceWorker from 'serviceWorker';
+
+const rootReducer = combineReducers({
```

Comment:
Not very sure about the set up of redux here. Could we try to utilise the Ducks pattern?
And have a `store.js` file at the root of the entire repository?